shared/stories/prop-providers.js

```diff
@@ -50,5 +52,25 @@ const Avatar = (following: string[], followers: string[]) => ({
   URLAvatar: (props: any) => props,
 })
 
+const TeamDropdownMenu = (adminTeams?: string[], teamMemberCounts?: {[key: string]: number}) => ({
```

Comment:
I might remove this from here and add it somewhere where the relevant stories can get to it